src/dlgTriggerEditor.cpp

```diff
@@ -125,68 +119,56 @@ dlgTriggerEditor::dlgTriggerEditor( Host * pH )
     setStatusBar( statusBar );
     statusBar->show();
     mIsGrabKey = false;
-    QVBoxLayout * pVB1 = new QVBoxLayout(mainArea);
+
+    // Replacement code to use single splitter for all right widgets:
+    mpSystemMessageArea = new dlgSystemMessageArea(splitter_rightTopBottom);
+    mpTriggersMainArea = new dlgTriggersMainArea(splitter_rightTopBottom);
+    mpTimersMainArea = new dlgTimersMainArea(splitter_rightTopBottom);
+    mpAliasMainArea = new dlgAliasMainArea(splitter_rightTopBottom);
+    mpActionsMainArea = new dlgActionMainArea(splitter_rightTopBottom);
+    mpKeysMainArea = new dlgKeysMainArea(splitter_rightTopBottom);
+    mpVarsMainArea = new dlgVarsMainArea(splitter_rightTopBottom);
+    mpScriptsMainArea = new dlgScriptsMainArea(splitter_rightTopBottom);
+    mpSourceEditorArea = new dlgSourceEditorArea(splitter_rightTopBottom);
+    mpErrorConsole = new TConsole(mpHost,false,splitter_rightTopBottom);
+    mpErrorConsole->setMinimumHeight( 200 );
+
+    //Prevent any region being "squashed down" i.e. taken below its minimum size
+    //so that it "disappears"
+    splitter_rightTopBottom->setChildrenCollapsible(false);
```

Comment:
Tricky because if things disappear I'm not sure if they will reappear when necessary.  E.g. the errors dialog (in the bottom right) could in the past be collapsed like that, and then clicking on the "Errors" button produced no obvious effect, i.e. it isn't showing, you click the button, and... nothing changes...

Would it be reasonable to make this total collapse behaviour dependant on the "Use Mudlet on a netbook with a small screen"?